### ISSUE TITLE
Pipelining Text Normalization

### ISSUE THREAD

**Issue by rdoume** [artefactory/NLPretext#35]
The idea is to transform the Nautilus_library using SK-learn compatible API, so we can use the Pipeline module of sklearn in order to chain the processing.
https://scikit-learn.org/stable/modules/generated/sklearn.pipeline.Pipeline.html
